assets/js/utils/get-shortcode.js

```diff
@@ -29,6 +29,9 @@ export default function getShortcode( { attributes }, name ) {
 		case 'woocommerce/product-best-sellers':
 			shortcodeAtts.set( 'best_selling', '1' );
 			break;
+		case 'woocommerce/product-top-rated':
+			shortcodeAtts.set( 'orderby', 'rating', '1' );
```

Comment:
```suggestion
			shortcodeAtts.set( 'orderby', 'rating');
```

You don't need the '1' here– this function sets the first param to the second in the shortcode, so this third parameter is not needed. In the other example, the shortcode is `best_selling="1"`, you can also see how we set the orderby a few lines ahead for the blocks that allow you to control that.